Mathlib/Data/Polynomial/Degree/Lemmas.lean

```diff
@@ -371,6 +377,24 @@ theorem natDegree_C_mul (a0 : a ≠ 0) : (C a * p).natDegree = p.natDegree := by
 set_option linter.uppercaseLean3 false in
 #align polynomial.nat_degree_C_mul Polynomial.natDegree_C_mul
 
+@[simp]
+lemma nextCoeff_C_mul_X_add_C (ha : a ≠ 0) (c : R) : nextCoeff (C a * X + C c) = c := by
+  rw [nextCoeff_of_natDegree_pos] <;> simp [ha]
+
+lemma natDegree_eq_one : p.natDegree = 1 ↔ ∃ a ≠ 0, ∃ b, C a * X + C b = p := by
```

Comment:
Here `b` can be replaced by `p.coeff 0`, giving a more precise statement (of course `a` is `p.coeff 1`, but I am not sure we want `p.natDegree = 1 ↔ p.coeff 1 ≠ 0 ∧ p = ...`

Reply:
Same here, this is for `subst` purposes.